src/main/application-update/tray-icons/checking-for-updates-tray-icon.injectable.ts

```diff
@@ -0,0 +1,34 @@
+/**
```

Comment:
Why are these here under and not under `tray/menu-icon` like the `normal-tray-icon`?